### ISSUE TITLE
bring back old theming options

### ISSUE THREAD

**Issue by imnotpoz** [lucasnlm/antimine-android#446]
none of the new themes really work for me, could you please bring back the old theming? I loved being able to configure each setting independently

**Comment by lucasnlm** [comment]
Hi, @jacekpoz !
Thanks for your feedback.
You mean you want to change the square shape and spaces like before? Or the colors?
I changed this because the old way was not efficient. But I can add an extra skin that fits to your needs if you let me know what you want it.

**Comment by jacekpoz** [comment]
>  You mean you want to change the square shape and spaces like before? Or the colors?

yeah, I'd love to be able to change all of the settings again - border radius, gap, color, etc.

> I changed this because the old way was not efficient.

oh, not efficient as in slow, or was the code ugly? in the first case I wouldn't mind personally, you probably wouldn't spend a lot of time configuring the theme, but I can't speak for everyone obviously

> But I can add an extra skin that fits to your needs if you let me know what you want it.

I'd need to go download an older apk and play around a bit, but I liked switching them around every once in a while, so you'd probably have to add a lot of new themes :P

**Comment by lucasnlm** [comment]
It was slower then the actual implementation. Specially in large maps it was too slow to render because every square was custom. The current implementation is way faster because it uses predefined sprites.

I tried to add "skins" to cover all possible personalization that was available before. So, I can add more skins based on your feedback.

**Comment by lucasnlm** [comment]
Here is an example of skin: https://github.com/lucasnlm/antimine-android/blob/main/gdx/src/main/assets/standard.png
You can try change it if you know how to edit pngs.

**Comment by jacekpoz** [comment]
sorry for the long wait, I was kinda busy recently

didn't know it'd have that big of a performance hit, that makes sense

and I don't think that adding new skins whenever someone asks would be that good, you'd end up with a lot of them which would in turn make the apk huge

not sure how hard it would be to bring back the old implementation, but maybe leaving it in the game with a warning about the bad performance would be a good idea? I liked to change the skin around pretty often and didn't see any performance drawbacks as I played on smaller map sizes

**Comment by SolsticeSpectrum** [comment]
Osu style skinning would be best to be honest, just have a specific folder where you can add custom Skins
